RecoParticleFlow/PFProducer/plugins/PFConcretePFCandidateProducer.cc

```diff
@@ -14,26 +14,32 @@ class PFConcretePFCandidateProducer : public edm::stream::EDProducer<> {
   ~PFConcretePFCandidateProducer() override;
 
   void produce(edm::Event&, const edm::EventSetup&) override;
+  static void fillDescriptions(edm::ConfigurationDescriptions& descriptions);
 
 private:
-  edm::InputTag inputColl_;
+  const edm::EDGetTokenT<reco::PFCandidateCollection> inputColl_;
 };
 
 DEFINE_FWK_MODULE(PFConcretePFCandidateProducer);
 
-PFConcretePFCandidateProducer::PFConcretePFCandidateProducer(const edm::ParameterSet& iConfig) {
-  inputColl_ = iConfig.getParameter<edm::InputTag>("src");
+void PFConcretePFCandidateProducer::fillDescriptions(edm::ConfigurationDescriptions& descriptions) {
+  edm::ParameterSetDescription desc;
+  desc.add<edm::InputTag>("src", {"particleFlow"});
+  descriptions.add("pfConcretePFCandidateProducer", desc);
```

Comment:
`addWithDefaultLabel` ?

Reply:
Addressed in a848a6c0ac22d23ae66d13f7c993f3c7ecdb1a15